include/revng/Model/ModelHelpers.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/IR/Function.h"
+
+#include "revng/Model/Binary.h"
+#include "revng/Support/Assert.h"
+#include "revng/Support/IRHelpers.h"
+#include "revng/Support/MetaAddress.h"
+
+namespace model {
+inline MetaAddress getMetaAddressOfIsolatedFunction(const llvm::Function &F) {
```

Comment:
We recently introduced a file for model-related helpers that deal with LLVM stuff: `include/revng/Model/IRHelpers.h`